### ISSUE TITLE
The session is not authenticated - timeout

### ISSUE THREAD

**Issue by ramonskie** [jetbrains-infra/packer-builder-vsphere#33]
if the packer script takes longer than 60 minutes the authentication seems to timeout for vsphere

```
2017/08/18 09:38:00 [INFO] (telemetry) ending powershell
1503049080,,ui,say,==> vsphere: Shut down VM...
1503049081,,ui,say,==> vsphere: Power off VM...
1503049086,,ui,error,==> vsphere: NotAuthenticated
1503049086,,ui,say,==> vsphere: Destroying VM...
1503049087,,ui,error,==> vsphere: ServerFaultCode: The session is not authenticated.
2017/08/18 09:38:07 [INFO] (telemetry) ending vsphere
2017/08/18 09:38:07 [INFO] (telemetry) found error: Cannot shut down VM: ServerFaultCode: The session is not authenticated.
1503049087,,ui,error,Build 'vsphere' errored: Cannot shut down VM: ServerFaultCode: The session is not authenticated.
1503049087,,error-count,1
2017/08/18 09:38:07 Builds completed. Waiting on interrupt barrier...
1503049087,,ui,error,\n==> Some builds didn't complete successfully and had errors:
1503049087,vsphere,error,Cannot shut down VM: ServerFaultCode: The session is not authenticated.
1503049087,,ui,error,--> vsphere: Cannot shut down VM: ServerFaultCode: The session is not authenticated.
1503049087,,ui,say,\n==> Builds finished but no artifacts were created.
2017/08/18 09:38:07 [WARN] (telemetry) Error finalizing report. This is safe to ignore. Post https://checkpoint-api.hashicorp.com/v1/telemetry/packer: context deadline exceeded
2017/08/18 09:38:07 waiting for all plugin processes to complete...
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /tmp/build/e55deab7/packer-builder-vsphere.linux: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
2017/08/18 09:38:07 /usr/local/bin/packer: plugin process exited
/tmp/build/e55deab7/bosh-windows-stemcell-builder/lib/stemcell/builder/base.rb:53:in `run_packer': Stemcell::Builder::PackerFailure (Stemcell::Builder::PackerFailure)
	from ci/tasks/create-vsphere-stemcell/create-vsphere-stemcell.rb:6:in `build'
	from ci/tasks/create-vsphere-stemcell/create-vsphere-stemcell.rb:64:in `<main>'
```

**Comment by ramonskie** [comment]
probably need to add roundtripper
see https://github.com/vmware/govmomi/issues/224

**Comment by mkuzmin** [comment]
Please test the fix from #30. Does it help?
